### DISCUSSION TITLE
Idea for making MDN Learning Area easier to grasp for beginners

### DISCUSSION THREAD

**Discussion by perborgen** [mdn/mdn-community#239] [Content]
Hi all 👋

My name is Per Harald Borgen, I am the founder of a code-learning platform called Scrimba. Earlier this week, I had an idea for how to make MDN articles easier to understand for beginners. So I [tweeted it out](https://twitter.com/perborgen/status/1573330047532175361), and it really resonated with people (200+ likes and dozens of comments from coding newbies).

The idea is basically to turn MDN images into [audiovisual scrims](https://scrimba.com/scrim/czvKPPsw) so that your visitors watch alternative explanations of your articles. You can see a [live demo of an MDN+scrim example here](https://aquamarine-unicorn-eee27f.netlify.app/#introducing_a_simple_example). 

_PS: it's important that you to click on the play icon in the scrim in order to watch the audiovisual explanation._

<a style="width: 50%;" href="https://aquamarine-unicorn-eee27f.netlify.app/#introducing_a_simple_example">
<img width="60%" alt="Screenshot 2022-09-25 at 13 01 02" src="https://user-images.githubusercontent.com/2429547/192162934-6997f69c-9126-4037-885b-c4dac5a0a9eb.png"></a>

Please note that the above example is NOT a video, it is a scrim. A scrim is a recorded coding session that requires 100 times less bandwidth than regular videos. So unlike videos, scrims are accessible for people in areas with slow internet access.

**Why it's important**

Many beginners find MDN too difficult, and resort to YouTube instead (e.g. [this tweet](https://twitter.com/scampbellatc/status/1573341328381210624)). Especially the new generation of coders seem to prefer _watching_ over _reading_ (3 out of 4 use YouTube according to freeCodeCamp's [survey](https://www.freecodecamp.org/news/the-2018-new-coder-survey-31-000-people-told-us-how-theyre-learning-to-code-and-getting-dev-jobs-e10feb9ed419/)). 

I am a big fan of MDN, and often send my students to the site, so I want it to be as good as possible, and stay relevant. So I would be more than willing to record scrims that you can embed into MDN. Meaning it won't be any work for Open Web Docs except approving the content and the PRs.

We have done the same with freeCodeCamp, as they wanted audiovisual explanations to support their curriculum. See example below 👇

<a href="https://www.freecodecamp.org/learn/responsive-web-design/css-flexbox/apply-the-flex-direction-property-to-create-rows-in-the-tweet-embed">
<img width="60%" alt="Screenshot 2022-09-25 at 13 37 01" src="https://user-images.githubusercontent.com/2429547/192164510-291ee5f0-e6ea-46d8-8aea-fc31c2e6dbfd.png">
</a>

Looking forward to hear what you all think of this idea 😊

**Comment by Josh-Cena** [comment]
FWIW I have mixed feelings about the entire learning area. IMO, intro to programming, intro to web developing, and intro to web technology X (X ∈ { JavaScript, MathML, CSS, ... }) have a world of difference in between, and MDN should really only take care of the latter two and assume readers have a decent understanding of how tech tutorials work already. Starting with programming 101 puts a huge burden on maintenance. But I don't touch learning docs anyway and I won't object.

**Comment by perborgen** [comment]
Thanks for the comment, @Josh-Cena! I agree totally that MDN should stick with teaching/documenting web development, so I am not argumenting for expanding to more areas. I am only talking about improving the content that is already there.

I don't agree that MDN should assume that Learning Area readers have a decent understanding of how tech tutorials work, as the Learning Area teaches _beginner-level_ web development. So it naturally attract newbies. 

I would have agreed with you had the Learning Area content been more advanced-level. But IMHO the more beginner-level tutorials MDN creates, the less they should assume about the readers' level of understanding.

**Reply by Josh-Cena** [reply]
That's true—given the de facto audience of the learning area, it makes sense to make them feel better. Just saying that "in an ideal world" we don't have to do this 😄 However, since we didn't make that assumption from the start, we are left with what we have.

**Reply by perborgen** [reply]
Yes, I think you're touching on something very important there: how it _feels_ for beginners to use the Learning Area. We want to make sure that no one leaves the MDN Learning Area feeling that they are stupid since they didn't grasp the article. At least not if they _could_ have understood it had the experience been richer 😊

**Reply by shantanuuchak** [reply]
Scrims specifically on the frontend path at MDN could be of great benefit.
